apps/settings/components/tab_content/index.styl

```diff
@@ -0,0 +1,5 @@
+.tab-content
+  margin unit auto
+
+  &__title
+    display none
```

Comment:
Where I'll handle the mobile stuff. This should really have been part of the actual tab component but I'm not going to try to merge any modifications to it.